pkgs/development/ocaml-modules/patricia-tree/default.nix

```diff
@@ -0,0 +1,61 @@
+{
+  lib,
+  buildDunePackage,
+  fetchFromGitHub,
+  findlib,
+  mdx,
+  qcheck-core,
+  re,
+  camlp-streams,
+  csexp,
+  astring,
+  ocaml-version,
+  fmt,
+  logs,
+  ppx_inline_test,
```

Comment:
```suggestion
  lib,
  buildDunePackage,
  fetchFromGitHub,
  mdx,
  qcheck-core,
  ppx_inline_test,
```

Reply:
The package does not build without these? I do not understand.